build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "spark-salesforce"
 
-version := "1.1.1"
+version := "1.1.2"
```

Comment:
Do not change the release version in the same PR. Create a separate PR for it. For our own release, we shouldn't upgrade the official version. Use `1.1.1-db1` in this case.